test/controllers/issues_controller_test.rb

```diff
@@ -18,4 +18,57 @@ class IssuesControllerTest < ActionDispatch::IntegrationTest
     assert_equal test_line_id, issue.line_onestop_id
     assert_response :success
   end
+
+  test "does not allow for visitor to create issue" do
+    test_stop_id = "1"
```

Comment:
Test data shouldn't be hard-coded, but should be pulled in from fixtures. I realize this precedent was set by the test above, but it would be better to do, `test_stop_id = stops(:one).id` for instance. That could even be put inline into the `get` call, making this test shorter and more readable.